library/common/glfw/embedder.cc

```diff
@@ -246,11 +246,13 @@ static void *GLFWProcResolver(void *user_data, const char *name) {
 //
 // Returns a caller-owned pointer to the engine.
 static FlutterEngine RunFlutterEngine(
-    GLFWwindow *window, const std::string &main_path,
-    const std::string &assets_path, const std::string &packages_path,
+    GLFWwindow *window, const std::string &assets_path,
     const std::string &icu_data_path,
     const std::vector<std::string> &arguments) {
-  std::vector<const char *> argv;
+  // FlutterProjectArgs is expecting a full argv, so when processing it for
+  // flags the first item treated as the executable and ignored. Add a dummy
```

Comment:
s/and/is ?

Reply:
Good catch; should have been "the first item **is** treated as the executable and ignored". Fixed in both files.